docs/admin-ui-meals.md

```diff
@@ -0,0 +1,136 @@
+# MealPlanner (GoodMealPlan) Website Documentation
+
+## Meals (Admin-Ui)
+
+### 1.1 Creating a Meal
+
+### Overview
+Creating a meal allows Admins to create new meals/recipes, which the users can view or add to their meal plan. 
+To create a meal, please follow the steps below:
+
+**Steps to Create a Meal**
+
+1. **Navigate to the Meal Plan Products Admin Page**
+    - Open your web browser and go to GoodMealPlan
+If you're not already logged in, sign in to your account
+Once logged in, you'll be directed to the new page from left pane select products list. Note: You must be an Admin to create a Meal.
+
+
+2. **Select the Meal tab.**
+    - Click the Meal tab on the top left sidebar of the admin-ui page of the MealPlanner app, similar to the screenshot below. 
+
+
+
+3. **Click the Create button**
+    - Click the `Create` button in the top right part of the admin-ui page. 
+    
+<img width="800" alt="image" src="create-meal.jpeg">
```

Comment:
Markdown image markup should look like this: ![Create Meal](create-meal.jpeg)

HTML style img tags won't work properly in the generated site.

Reply:
Thank you! I have made the changes to fix the image addition into the md file.